src/main/resources/edu/cornell/kfs/module/cam/businessobject/datadictionary/Asset.xml

```diff
@@ -82,27 +85,28 @@
       <property name="beanName" value="Asset-lookupDefinition" />
       <property name="fieldOverrides">
           <list>
+              <!-- NEW FRAMEWORK -->

```

Comment:
In this release, Asset lookups appear to have been configured for both the old and new frameworks in base code. Instead of repurposing the old-lookup overrides for the new framework, I would suggest creating new field override beans to put these changes in the new framework.

Reply:
Fixed.